eks-worker-al2.json

```diff
@@ -71,7 +71,7 @@
         {
           "device_name": "/dev/xvda",
           "volume_type": "gp2",
-          "volume_size": 20,
+          "volume_size": 512,
```

Comment:
Will this be a lot more expensive, or do we get 512 included on these instances?

Reply:
original images had 512 disk, we can make it 256 or 128.